Assets/MixedRealityToolkit.Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -117,13 +148,28 @@ private void ApplySettingsForOpaqueDisplay()
         /// </summary>
         private void ApplySettingsForTransparentDisplay()
         {
-            CameraCache.Main.clearFlags = CameraProfile.CameraClearFlagsTransparentDisplay;
-            CameraCache.Main.backgroundColor = CameraProfile.BackgroundColorTransparentDisplay;
-            CameraCache.Main.nearClipPlane = CameraProfile.NearClipPlaneTransparentDisplay;
-            CameraCache.Main.farClipPlane = CameraProfile.FarClipPlaneTransparentDisplay;
+            Main.clearFlags = CameraProfile.CameraClearFlagsTransparentDisplay;
+            Main.backgroundColor = CameraProfile.BackgroundColorTransparentDisplay;
+            Main.nearClipPlane = CameraProfile.NearClipPlaneTransparentDisplay;
+            Main.farClipPlane = CameraProfile.FarClipPlaneTransparentDisplay;
             QualitySettings.SetQualityLevel(CameraProfile.HoloLensQualityLevel, false);
         }
 
+        /// <summary>
+        /// Finds or creates the main camera.
+        /// </summary>
+        private void FindOrCreateMainCamera()
+        {
+            // If the cached camera is null, search for main
+            main = Camera.main;
+
+            if (main == null || !main.isActiveAndEnabled)
+            {   // If no main camera was found, create it now
+                Debug.LogWarning("No main camera found. The Mixed Reality Toolkit requires at least one camera in the scene. One will be generated now.");
+                main = new GameObject("Main Camera", typeof(Camera)) { tag = "MainCamera" }.GetComponent<Camera>();
+            }
```

Comment:
Forcing a camera creation seems like it could really disrupt a scene if somehow this function is called while still configuring a scene. I think it may be better practice to document that the camera may be null.

Reply:
Let's discuss in #4896